packages/components/multiselect/examples/MultiselectReferenceExample.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Button, Stack } from '@contentful/f36-components';
+import { Multiselect } from '@contentful/f36-multiselect';
+
+export default function MultiselectReferenceExample() {
+  const spaces = [
+    'Travel Blog',
+    'Finnance Blog',
+    'Fitness App',
+    'News Website',
+    'eCommerce Catalogue',
+    'Photo Gallery',
+  ];
+
+  const [selectedItems, setSelectedItems] = React.useState([]);
+  const [filteredItems, setFilteredItems] = React.useState(spaces);
+  const togglePopOverRef = React.useRef(null);
+  const clearSearchFieldRef = React.useRef(null);
+
+  const handleSearchValueChange = (event) => {
+    const value = event.target.value;
+    const newFilteredItems = spaces.filter((item) =>
+      item.toLowerCase().includes(value.toLowerCase()),
+    );
+    setFilteredItems(newFilteredItems);
+  };
+
+  const handleSelectItem = (event) => {
+    const { checked, value } = event.target;
+    if (checked) {
+      setSelectedItems((prevState) => [...prevState, value]);
+    } else {
+      const newSelectedFruits = selectedItems.filter(
```

Comment:
[nit] why are we calling them fruits? There's nothing about fruits in this file

Reply:
ah, sorry, copy paste issue 😅